packages/sdk-components-react/src/text.tsx

```diff
@@ -9,7 +9,9 @@ type Props = ComponentProps<typeof defaultTag> & {
 };
 
 export const Text = forwardRef<ElementRef<typeof defaultTag>, Props>(
-  ({ tag: Tag = defaultTag, children, ...props }, ref) => {
+  ({ tag = defaultTag, children, ...props }, ref) => {
+    // Can't map it in the destricturing, default type won't be generated correctly
+    const Tag = tag;
```

Comment:
how does it work now without this change?

Reply:
it ignores default value if you add the mapping to the argument ...

Reply:
My guess is that someone chagned it to use destructuring/maping and didn't rebuild or we upgraded the compiler library and it broke